src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -553,8 +550,7 @@ public void removeDatabaseChangeListener(DatabaseChangeListener l) {
      */
     public static String getResolvedField(String field, BibEntry entry, BibDatabase database) {
         if ("bibtextype".equals(field)) {
-            TypedBibEntry typedEntry = new TypedBibEntry(entry, Optional.ofNullable(database));
-            return typedEntry.getTypeForDisplay();
+            return EntryUtil.capitalizeFirst(entry.getType());
```

Comment:
Is problematic as this method is called from a lot of places where I recall we do not have a LoadedDatabase.